sympy/physics/units/unitsystem.py

```diff
@@ -192,3 +199,9 @@ def _collect_factor_and_dimension(self, expr):
             return S.One, expr
         else:
             return expr, Dimension(1)
+
+    def get_units_non_prefixed(self) -> tSet[Quantity]:
```

Comment:
Maybe this could be private, i.e. start with an underscore `_` character. Do end-users usually need to access this method?

Reply:
OK.